openslides_backend/models/base.py

```diff
@@ -138,12 +138,34 @@ def get_required_fields(self) -> Iterable[fields.Field]:
         for model_field in self.get_fields():
             if model_field.required:
 
-                if (
-                    isinstance(model_field, fields.RelationListField)
-                    or isinstance(model_field, fields.GenericRelationListField)
-                    or isinstance(model_field, fields.BaseTemplateField)
+                if isinstance(
+                    model_field,
+                    (
+                        fields.RelationListField,
+                        fields.GenericRelationListField,
+                        fields.BaseTemplateField,
+                    ),
                 ):
                     raise NotImplementedError(
                         f"{self.collection.collection}.{model_field.own_field_name}"
                     )
                 yield model_field
+
+    @staticmethod
+    def check_required_not_fulfilled(
+        field: fields.Field, instance: Dict[str, Any]
+    ) -> bool:
+        """
+        Preconditions:
+        - This method depends on the types eliminated by te get_required_fields method above
+        - The instance-dict contain field.own_field_name as key
+        It returns True if the related value in instance doesn't fulfill the require-requirement
+        """
+        if field.own_field_name not in instance:
+            return False
+        if isinstance(
+            field, (fields.IntegerField, fields.BooleanField, fields.FloatField)
+        ):
+            return instance[field.own_field_name] is None
+        else:
+            return not instance[field.own_field_name]
```

Comment:
This method belongs to the `Field`, not the `Model`, and it should suffice to just pass the value of the field instead of the full instance. I would suggest to move this method to the base `Field` class and overwrite it (either directly or via a mixin) in `IntegerField`, `BooleanField` and `FloatField`. That way, it is also easily adjustable on a per-field basis.

Reply:
Good idea, also it needs instance and create/update information => Done